gamehub-backend/src/test/java/com/example/gamehubbackend/services/IdServiceUnitTest.java

```diff
@@ -0,0 +1,19 @@
+package com.example.gamehubbackend.services;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class IdServiceUnitTest {
+
+    IdService idService = new IdService();
+
+    @Test
+    void testRandomId() {
+        String id = idService.randomId();
```

Comment:
I think you'd actually need to mock the UUID class (to not have a dependency), like so:

```
        UUID uuid = UUID.randomUUID();
        try (MockedStatic<UUID> mockedUUID = mockStatic(UUID.class)) {
            mockedUUID.when(UUID::randomUUID).thenReturn(uuid);
            //WHEN
            String actual = idService.randomId();
            //THEN
            String expected = uuid.toString();
            mockedUUID.verify(UUID::randomUUID);
            assertEquals(expected, actual);
        }
```
but your version is also pretty cool!

Reply:
fixed